Library/Homebrew/utils/sorbet.rb

```diff
@@ -0,0 +1,8 @@
+# typed: strict
+# frozen_string_literal: true
+
+if ENV["HOMEBREW_TESTS_COVERAGE"]
```

Comment:
Why `TESTS_COVERAGE` specifically? Do you think this warrants a separate environment variable (that we could document, too)?

Reply:
There was no `HOMEBREW_TESTS` variable and I think `HOMEBREW_TESTS_COVERAGE` implies we run everything possible. Of course we can add another variable if needed, but I think it's enough for now.

Reply:
`HOMEBREW_TESTS_COVERAGE` implies `--coverage` has been passed. I think a dedicated `HOMEBREW_SORBET_RUNTIME` documented variable in `EnvConfig` and set by e.g. `brew tests` would be good 👍🏻